### PR TITLE
Adding reporting metric for datapoints requested via http endpoint

### PR DESCRIPTION
**Pull request by vibhory2j**
Adding reporting metric for datapoints requested via http endpoint. The metric can be queried as `kairosdb.http.datapoints_requested`. The count of datapoints requested provided is before their aggregation.

### CODE REVIEW

**Comment by vibhory2j** [comment]
Addresses #71

**Comment by vibhory2j** [comment]
Tested on staging and seems to be working fine.

File: src/main/java/org/kairosdb/core/http/rest/MetricsResource.java [hunk 1]
```diff
@@ -661,20 +664,28 @@ private Response executeNameQuery(NameType type) {
...
+		dpsRequestedCount.addTag("deployment_id", m_deploymentId);
 
 		dpsCount.addDataPoint(m_longDataPointFactory.createDataPoint(now, count));
 		dpsTime.addDataPoint(m_longDataPointFactory.createDataPoint(now, time));
+		dpsRequestedCount.addDataPoint(m_longDataPointFactory.createDataPoint(now, m_datapointsCount));
 		List<DataPointSet> ret = new ArrayList<DataPointSet>();
-		ret.add(dpsCount);
-		ret.add(dpsTime);
+
+		if (count == 0){
```
Comment (rajatparida86) [inline src/main/java/org/kairosdb/core/http/rest/MetricsResource.java]: Anyways you are going to add the `dpsRequestedCount` irrespective of `count` So you can replace with:
```
ret.add(dpsRequestedCount);
	if (count != 0){
 			ret.add(dpsCount);
 			ret.add(dpsTime);
 	}
	return ret;
```

**Review by rajatparida86 — CHANGES_REQUESTED** [review]

**Review by rajatparida86 — APPROVED** [review]

**Comment by rajatparida86** [comment]
👍

**Comment by vibhory2j** [comment]
👍

**Pull request merged**